.github/workflows/ci.yml

```
@@ -83,11 +83,34 @@ jobs:
         name: ${{ matrix.os.name }} - ${{ matrix.python.name }}
         path: output_post.log
 
-    - name: Test
+    - name: Test - post
       shell: bash
       run: |
         python test.py --path output_post.log --package attrs 20.3.0
 
+    - name: Create env and install requests
+      shell: bash
+      run: |
+        python -m pip install virtualenv
+        python -m virtualenv -p python an_env
+        an_env/*/python -m pip install six==1.15.0
+
+    - name: Run the action for the env
+      uses: ./
+      with:
+        output-path: output_env.log
+        python-path: an_env/*/python
```

Instead of `python-path` can we have `python-env-path` ... or something like that?
And then pyton-info-action can apply the right path based on OS version.

Now I understand that with pypy vs py3.7 or py3.8 this can be more complicated.

I guess that we can start with the python-path .. and see how it goes

```suggestion
        python-bin-path: an_env/*/python
```

Not sure if `python-bin-path` or `python-exe-path` ... or `python-binary-path` ... or `python-lauch-path`

---

So I'll at least write out my previous counter argument here.  Taking the env directory is less flexible and places more responsibility on this action and it's not clear what it makes easier given that `env/*/python` works in all tested cases (3 os, 2 interpreter, multiple interpeter versions).  Unless maybe tox induces different behavior than direct env creation so maybe I should add a test for that as well.

Any chance you have a case where `an_env/*/python` doesn't work?

Let's take a stab at documentation.

> If set, `python-env-path` will be taken as the path to the virtual environment directory.  That is the directory that contains either `bin/python` or `Scripts/python` depending on the OS and Python interpreter.

vs.

> If set, `python-path` will be used as the path to the Python executable.  Note that `an_env/*/python` expands properly in most cases when using a virtual environment.

I dunno.  I don't wanna be bullheaded but taking the env path mostly seems to me like something that feels simpler but isn't actually.

---

Thanks for the info. I am happy to use python-path :)